services/src/handlers/datasets.rs

```diff
@@ -231,6 +260,34 @@ impl<C: Context> FromRequest for AdminOrSession<C> {
     }
 }
 
+#[derive(Debug, Snafu, IntoStaticStr)]
+#[snafu(visibility(pub(crate)))]
```

Comment:
- why does it have this visibility?
- use "#[snafu(context(suffix(false)))] // disables default `Snafu` suffix" like on our other error types

Reply:
It needs to be visible in the entire crate because it is also used in the pro create_dataset_handler.

Reply:
[b4e7866](https://github.com/geo-engine/geoengine/pull/751/commits/b4e7866f3d96466ef8cd8a66d0aef78ac50a815f)